app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script.rb

```diff
@@ -29,10 +29,14 @@ def raw_delete_in_provider
   end
 
   def run(vars = {})
-    workflow      = ManageIQ::Providers::AnsiblePlaybookWorkflow
-    extra_vars    = merge_extra_vars(vars[:extra_vars])
-    playbook_vars = { :playbook_path => parent.path }
-    credentials   = collect_credentials(vars)
+    workflow = ManageIQ::Providers::AnsiblePlaybookWorkflow
+
+    extra_vars = merge_extra_vars(vars[:extra_vars])
+
+    checkout_dir  = checkout_git_repository # TODO: what will cleanup this dir?
```

Comment:
Could you add a state to the [workflow](https://github.com/ManageIQ/manageiq/blob/5882eb24a14031cacd7ab2ff9cf3579c4500fdd3/app/models/manageiq/providers/ansible_playbook_workflow.rb) to cleanup the repo?

Reply:
I can, but it would have to be optional based on a parameter or something.  AnsiblePlaybookWorkflow was written expecting the directory to be owned outside of it (i.e. playbooks stored in the manageiq plugins themselves).

Reply:
I was going to do that in a followup.

Reply:
:+1: